app/src/main/res/layout/item_list.xml

```diff
@@ -20,7 +22,15 @@
         app:srcCompat="@drawable/ic_drag_vertical_white_24dp"
         tools:ignore="ContentDescription" />
 
+    <View
```

Comment:
I plan on going through this and documenting everything tonight, but I'll definitely do this.